.github/workflows/_ci.yaml

```diff
@@ -129,10 +113,55 @@ jobs:
       BASE_LIBRARY: pax
     secrets: inherit
 
+  save-tags:
+    runs-on: ubuntu-22.04
+    needs: [build-jax, build-t5x, build-pallas, build-pax, build-rosetta-t5x, build-rosetta-pax]
+    outputs:
+      ALL_TAGS: ${{ steps.save-docker-tags.outputs.ALL_TAGS }}
+    steps:
+      - name: Save docker tags in a single JSON object
+        id: save-docker-tags
+        run: |
+          ALL_TAG_JSON=$(cat << EOF
+            { \
+            "TAG_JAX_MEALKIT": "${{ needs.build-jax.outputs.DOCKER_TAG_MEALKIT }}", \
+            "TAG_JAX_FINAL": "${{ needs.build-jax.outputs.DOCKER_TAG_FINAL }}", \
+            "TAG_UPSTREAM_T5X_MEALKIT": "${{ needs.build-t5x.outputs.DOCKER_TAG_MEALKIT }}", \
+            "TAG_UPSTREAM_T5X_FINAL": "${{ needs.build-t5x.outputs.DOCKER_TAG_FINAL }}", \
+            "TAG_PALLAS_MEALKIT": "${{ needs.build-pallas.outputs.DOCKER_TAG_MEALKIT }}", \
```

Comment:
Is this consistent with `_build.yaml`:
```
echo "TAG_${CONTAINER_TAG_NAME}_MEALKIT=${{ steps.mealkit-metadata.outputs.tags }}" | tee -a $GITHUB_OUTPUT
```
should it be
```
"TAG_PALLAS_MEALKIT": "${{ needs.build-pallas.outputs.TAG_PALLAS_MEALKIT}}"
```
?

Reply:
Yes. See comments above.